drizzlepac/photeq.py

```diff
@@ -133,8 +128,8 @@ def photeq(files='*_flt.fits', sciext='SCI', errext='ERR',
 
         .. note::
 
-            If specifying multiple secondary photometric keywords in the TEAL
```

Comment:
Here is where I might have left the comment and added some text indicating instruction may be removed in the future.